indicators/NO_FUNC_001-004/R/code Joachim.R

```diff
@@ -0,0 +1,211 @@
+We can calculate national and region-wise means and their related standard errors. But note that calculating a simple mean would be inappropriate for these data. This is because:
```

Comment:
This file should be removed if not needed or moved to another location (less visible).